src/dsymbol/conversion/second.d

```diff
@@ -298,6 +301,53 @@ body
 	}
 }
 
+unittest
```

Comment:
Why don't you use the existing testing system ? It's more compact and does the same.